README.md

```diff
@@ -45,6 +45,14 @@ To allow access to Docker containers you would do:
 -A DOCKER_INPUT -s 192.168.0.1/32 -j DOCKER
 ```
 
+The `$accept_rules` parameter for the main `docker_firewall` class provides an easy way to set up rules to accept connections to Docker containers. This parameter takes a hash of `firewall` resources to create, and defaults those rules to be in the `DOCKER_INPUT` chain and jump to the `DOCKER` chain. This parameter is probably best set using Hiera, for example:
+```yaml
+docker_firewall::accept_rules:
+  200 accept all traffic from 192.168.0.1/32:
```

Comment:
YAML quoting rules confuse me, so I generally quote strings fairly often. I'm happy with this if it works, though.